npm/react/examples/a11y/cypress/plugins/index.js

```diff
@@ -1,9 +1,14 @@
-// load file preprocessor that comes with this plugin
+// @ts-check
+
+// load file setupDevServer that comes with this plugin
 // https://github.com/bahmutov/cypress-react-unit-test#install
-const preprocessor = require('@cypress/react/plugins/react-scripts')
+const setupDevServer = require('@cypress/react/plugins/react-scripts')
```

Comment:
Can we call the default export function `devServer` in all the dev server examples and READMEs? I see a bunch of `setup*` and also `inject*` names throughout this PR.

Reply:
done in c1f9bbc

Reply:
You seem to have missed this file